packages/core/src/config.js

```diff
@@ -44,6 +44,10 @@ export const configSchema = {
         type: 'boolean',
         default: false
       },
+      enableLayout: {
+        type: 'boolean',
+        default: false
```

Comment:
```suggestion
```

lets not add default for now. we can interpret it as false when nothing is sent.

Reply:
removed default value.